src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml

```diff
@@ -197,7 +197,23 @@
 
         <!-- Template for the feedback banner -->
         <DockPanel Name="Feedback" VerticalAlignment="Bottom" Grid.Row="2" Width="600">
-            <StatusBar Background="{StaticResource FeedbackSectionBackground}" DockPanel.Dock="Bottom" Height="55">
+            <!-- Restart Banner -->
```

Comment:
I'm curious why you all decided to copy the preview banner UI for this instead of adding another state icon. Seems like we would want to avoid conflating dependency-state and feature-state UI components.

Reply:
@scottmitchell The team have discussed around this state, we admit this is not a state for package like the other ones, this is more like a state for Dynamo. So we decide to display this differently and still show it as a warning.